brayns/common/PropertyMap.h

```diff
@@ -353,9 +358,19 @@ class PropertyMap
 
     /** @return all the registered properties. */
     const auto& getProperties() const { return _properties; }
-    /** Merge this property map with properties from another. */
+    /** Merge this property map with properties from another.
+     *  @throw std::runtime error if a property with the same name but
+     *  incompatible types is found in both maps.
+     */
     void merge(const PropertyMap& input);
 
+    /** Take the values from another property map only for properties that
+     *  are known and compatible to this one.
+     *  @throw std::runtime error if a property with the same name but
+     *  incompatible types is found in both maps.
+     */
+    void take(const PropertyMap& input);
```

Comment:
Maybe we should call these takeAll and takeExisting to make the difference between the two methods clearer. They only differ in an else clause anyway.

Reply:
I prefer updateExisting rather than takeExisting and leave merge as merge.